### PR TITLE
Add Actuator Panel

### PR DESCRIPTION
**Pull request by magicrub**
This adds an Actuator Panel which looks very similar to the ESC Panel.

image

### CODE REVIEW

**Comment by pavel-kirienko** [comment]
Hi Tom!

It would appear that @polarisru is also working on a similar thing concurrently:

image

Alex, would you like to provide any input here or should I just merge it right away?

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
This codebase is slowly falling into a state of mild deterioration because the UAVCAN dev team has been mostly focused on [releasing the stable version of the protocol and building the ecosystem around it](https://forum.uavcan.org/t/2020-recap-and-roadmap/1051). This tool has never been intended as a long-term solution so it's missing key components necessary for any well-maintained project like well-defined policies on accepting contributions and releasing new versions. All this means that I am going to accept this changeset (once the minor issues I highlighted here are addressed) without much testing and I am not sure if I am going to be able to build and publish new release binaries with this changeset in place.

**Comment by Polarisru** [comment]
Yes, you are right :)
Here is my actuator panel looks like:
actuator
It's up to you, I also will add some product-specific extra features there, but in another panel, this one stays compatible with UAVCANv0 for actuators.

**Comment by pavel-kirienko** [comment]
Okay. I am inclined to prefer Tom's version then, if only because he got here first.

**Comment by Polarisru** [comment]
> 
> 
> Okay. I am inclined to prefer Tom's version then, if only because he got here first.

Ok, no problem, but my version has position readout functionality

**Comment by pavel-kirienko** [comment]
I don't think it's that important either way because UAVCAN v0 in general and this tool in particular are not expected to stay relevant for much longer.

**Comment by Polarisru** [comment]
Ok, we have a lot of UAVCAN v0 servos already and have to support them, so I will continue to improve this tool anyway.

File: uavcan_gui_tool/panels/acturator_panel.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
+#
+# Copyright (C) 2021  UAVCAN Development Team  <uavcan.org>
+#
+# This software is distributed under the terms of the MIT License.
+#
+# Author: Tom Ptttenger <magicrub@gmail.com>
```
[the remark this answers is not in the document]
Comment (magicrub) [reply uavcan_gui_tool/panels/acturator_panel.py]: how the hell did I spell my name wrong?!?!?!? Thanks for finding this

File: uavcan_gui_tool/panels/__init__.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 # TODO: Load all inner modules automatically. This is not really easy because we have to support freezing.
 from . import esc_panel
+from . import acturator_panel
```
[the remark this answers is not in the document]
Comment (magicrub) [reply uavcan_gui_tool/panels/__init__.py]: fixed

**Review by magicrub — COMMENTED** [review]
Thanks for finding those. They were embarrassing enough that I squash it into the same commit :)

**Comment by magicrub** [comment]
fixed up. Thanks for the review @pavel-kirienko

**Comment by magicrub** [comment]
Also, the latest Windows release is quite old. It doesn't support custom dsdl. I would appreciate a new release pretty please.

**Review by pavel-kirienko — APPROVED** [review]

**Comment by davidbitton** [comment]
do these changes propagate to the pip installed version?

**Comment by pavel-kirienko** [comment]
No. I might look into that next week.

**Pull request merged**